src/components/Header/header.css

```diff
@@ -3,6 +3,7 @@
   top: 0;
   left: 0;
   width: 100%;
+  height: 45px;
```

Comment:
I set the height here because when the DropDown would render in the header, the header height would shrink, leaving a space between the header and the content.

 Setting the height of the header-container to 45px really causes the element to be 57px high due to the padding. It is now consistent across browsers and screen sizes.

 I'm not sure if we should leave headerHeight in app.jsx as a piece of state that can be updated depending on what we do in the future, or if we should just explicitly set the height like we did with FOOTER_HEIGHT.

But This works just fine for now.

Reply:
I don't really have enough experience in this to give any input here so this is good if it works for now but yea just keep this in mind as you learn more and stuff and maybe we'll eventually change it to what you're thinking